backend/server.py

```
@@ -6431,6 +6453,54 @@ async def get_public_doctor() -> _DoctorReport:
+    site / public Doctor view can show a live read-only KPI strip (brief #6).
+    Only non-sensitive process-wide counts are exposed — no user, company, repo,
+    or task identifiers. Counters are cumulative since process start
+    (``uptime_seconds`` gives the window); fields that are not yet instrumented
+    are reported honestly rather than fabricated.
+    """
+    import datetime
+
+    try:
+        from agent.kpi import get_tracker
+        snapshot = get_tracker().snapshot().as_dict()
+    except Exception as exc:  # pragma: no cover - defensive
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)